apps/fz_http/lib/fz_http_web/live/setting_live/account.html.heex

```diff
@@ -9,6 +9,12 @@
       form: "account-edit") %>
 <% end %>
 
+<%= if @live_action == :register_mfa do %>
```

Comment:
We'll probably need this in the `unprivileged` user's section as well. Will eventually need some refactoring around how we contextualize views for the `admin` and `unprivileged` portals at some point.

Reply:
Yep 😄 it's planned. Was going to get the workflow reviewed first, as there is more space (literally) for new functionality on admin section, easier to add first. Planning to add a new page for unprivileged users, not enough screen space for more functionality there.